flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX <= 0)
```

Comment:
I changed the comments, so it is easier to be understood.
Just think if somehow a negative value is passed in, `FlxGame` can deal with it.